buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -26,15 +26,14 @@ object Versions {
   const val materialVersion = "1.1.0"
   const val coroutinesVersion = "1.4.3"
 
-  const val testCoreVersion = "1.2.0"
-  const val junitVersion = "4.13"
-  const val junitTestExtVersion = "1.1.2"
+  const val testCoreVersion = "1.4.0"
+  const val junitVersion = "4.13.2"
+  const val junitTestExtVersion = "1.1.3"
   const val truthVersion = "1.0"
   const val mockitoVersion = "2.23.4"
-  const val testRunnerVersion = "1.0.2"
+  const val testRunnerVersion = "1.4.0"
   const val uiAutomatorVersion = "2.2.0"
-  const val extVersion = "1.1.1"
-  const val espressoVersion = "3.0.2"
+  const val espressoVersion = "3.4.0"
```

Comment:
this library pulls in BootstrapActivity, EmptyActivity, and EmptyFloatingActivity. It's important to use newer versions of the lib so that we get this commit: https://github.com/android/android-test/commit/dd6df4aa22b77290fbe45834619217fd4852ba05